src/chains/ethereum/ethereum/tests/api/debug/debug_storageRangeAt.test.ts

```diff
@@ -134,141 +147,108 @@ describe("api", () => {
 
       it("should return account doesn't exist error when debugging a deployment transaction", async () => {
```

Comment:
```suggestion
      it("should return "account doesn't exist" error when debugging a deployment transaction", async () => {
```
pls

Reply:
this would breaks the things.

Reply:
Ahh, you're so right. Reading that `it` statement took me about 5 attempts to understand it, but based off of my other review suggestions from that day, I think I was a little slow when I reviewed this :sweat_smile: